gui/slick/interfaces/default/manage_episodeStatuses.tmpl

```diff
@@ -40,50 +38,49 @@ Manage episodes with status <select name="whichStatus" class="form-control form-
 <form action="$sbRoot/manage/changeEpisodeStatuses" method="post">
 <input type="hidden" id="oldStatus" name="oldStatus" value="$whichStatus" />
 
-<h2><span class="grey-text">Shows containing</span> $common.statusStrings[$int($whichStatus)] <span class="grey-text">episodes</span></h2>
-
-<br />
+<h3>${len($sorted_show_ids)} Shows containing <span class="grey-text">$common.statusStrings[$int($whichStatus)]</span> episodes</h3>
 
 #if $whichStatus in ($common.ARCHIVED, $common.IGNORED, $common.SNATCHED):
-#set $row_class = "good"
+	#set $row_class = "good"
 #else
-#set $row_class = $common.Overview.overviewStrings[$whichStatus]
+	#set $row_class = $common.Overview.overviewStrings[$whichStatus]
 #end if
 <input type="hidden" id="row_class" value="$row_class" />
 
-Set checked shows/episodes to <select name="newStatus" class="form-control form-control-inline input-sm">
-#set $statusList = [$common.SKIPPED, $common.WANTED, $common.ARCHIVED, $common.IGNORED]
-#if $int($whichStatus) in $statusList
-$statusList.remove($int($whichStatus))
-#end if
-
-#if $int($whichStatus) in [$common.SNATCHED, $common.SNATCHED_PROPER]
-$statusList.append($common.FAILED)
-#end if
-
-#for $curStatus in $statusList:
-<option value="$curStatus">$common.statusStrings[$curStatus]</option>
-#end for
-</select>
-<input class="btn btn-inline go" type="submit" value="Go" />
+<div class="form-group">
+	Set checked shows/episodes to <select name="newStatus" class="form-control form-control-inline input-sm">
+	#set $statusList = [$common.SKIPPED, $common.WANTED, $common.ARCHIVED, $common.IGNORED]
```

Comment:
the previous line of python at L#47 was an #end if at column 1, therefore, this python line should be at column 1, so to the next line, and the line after that should be indented by 1 set(of spaces) not tabs, etc